docs/dashboards/your-first-dashboard.md

```diff
@@ -1,24 +1,24 @@
 Your first dashboard
 ===
 
-If you are ready to create your first simple dashboard please follow the steps:
+When you are ready to create your first simple dashboard, please follow these steps:
 
 1. Add a div to your webpage with a specific id.
 
     ```html
         <div id="container">
     ```
 
-2. In this basic example we are going to create a dashboard with two cells. One will contain a static HTML and the other will contain a chart created with the Highcharts.
-Thus we have to import the main dashboard package, the Highcharts library and a plugin that allows implementing the Highcharts chart inside the dashboard.
+2. In this basic example we are going to create a dashboard with two cells. One will contain a static HTML and the other will contain a chart created with Highcharts.
+Thus, we have to import the main dashboard package, the Highcharts library and a plugin that allows implementing the Highcharts chart inside the dashboard.
```

Comment:
There should be no comma.

Reply:
It's the input from Lars Helge.

Reply:
Same as for _therefore_ at the beginning of a sentence.